### DISCUSSION TITLE
Using standard system documentation about ambiguous conditions

### DISCUSSION THREAD

**Discussion by ruv** [ForthHub/discussion#154] [Polls]
There is a [proposal](https://forth-standard.org/proposals/relax-documentation-requirements-of-ambiguous-conditions) to slightly relax requirements for a standard system documentation concerning behavior in ambiguous conditions.

Namely, this proposal suggests not to **require** a standard Forth system to document behavior in ambiguous conditions, but only to **recommend** to do that.

Some examples of such documentation:
- SwiftForth: in the installation directory `doc/swiftforth-linux-macos.pdf` / Appendix B: Standard Forth Documentation Requirements / Table 16: SwiftForth action on ambiguous conditions.
- Gforth: [documentation online](https://gforth.org/manual/core_002dambcond.html).
- SP-Forth: [documentation online](https://spf.sourceforge.net/docs/ans.en.html#amb).

Please share links to other Forth systems that have this documentation.

So, there is a question for **users** of Forth systems.

**Comment by mhx-gh** [comment]
> Please share links to other Forth systems that have this documentation.
In iForth: <installpath>/dfwforth/ifdocs/
